pages/api/trees/plantTree.js

```diff
@@ -30,6 +30,7 @@ export default function handler(req, res) {
 
   fetch("https://api.sandbox.digitalhumani.com/tree", requestOptions)
     .then((response) => {
+      console.log("Tree planted!");
```

Comment:
left console.log

Reply:
meh, leave it in, i quite like it when testing